scripts/deploy.sh

```diff
@@ -0,0 +1,12 @@
+# This script runs the aws cli command to deploy the app to s3
+# It also uploads source maps to bugsnag
+
+# Try this step and throw an error if it fails
+echo "Deploying to S3"
+aws s3 sync build/ s3://${BUCKET}/ --acl public-read --follow-symlinks --delete --exclude .cmdrc.json 
+echo "Deployed to S3"
```

Comment:
Oops sorry, I think this actually might be
```suggestion
aws s3 sync build/ s3://${BUCKET}/ --acl public-read --follow-symlinks --delete --exclude env/.cmdrc.json 
echo "Deployed to S3"
```

Reply:
Don't need it actually the file doesn't exist in the build directory anymore.